admin_manual/configuration/user/user_provisioning_api.rst

```diff
@@ -345,20 +434,29 @@ XML Output
     </meta>
     <data/>
   </ocs>
-  
-**users / createsubadmin**
---------------------------
 
-Makes a user the subadmin of a group. Authentication is done by sending a Basic 
-HTTP Authorization header.
+Create Sub-admin
+----------------
+
+Makes a user the sub-admin of a group.
+Authentication is done by sending a Basic HTTP Authorization header.
 
-**Syntax: ocs/v1.php/cloud/users/{userid}/subadmins**
+============================================== ======== ==============
+Request Path                                   Method   Content Type
+============================================== ======== ==============
+``ocs/v1.php/cloud/users/{userid}/sub-admins`` ``POST`` ``text/plain``
+============================================== ======== ==============
 
-* HTTP method: POST
-* POST argument: groupid, string - the group of which to make the user a 
-  subadmin
+======== ====== =========================================
+Argument Type   Description
+======== ====== =========================================
+userid   string The id of the user to retrieve groups for
```

Comment:
Description not correct. Maybe:
the id of the user who is to be made a subadmin